pkgs/tools/misc/fastfetch/default.nix

```
@@ -0,0 +1,23 @@
+{ cmake, fetchFromGitHub, lib, pkgconfig, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "fastfetch";
+  version = "1.5.2";
```

```suggestion
  version = "1.5.3";
```
Might as well change to the newest version since it came out on the same date.

---

Done